src/main/java/interface_adapter/translateText/TranslateTextPresenter.java

```diff
@@ -19,8 +19,9 @@ public TranslateTextPresenter(TranslateTextViewModel translateTextViewModel) {
      * @param translatedText the output data
      */
     @Override
-    public void prepareSuccessView(String translatedText, String inputLanguage) {
+    public void prepareSuccessView(String inputText, String translatedText, String inputLanguage) {
         translateTextViewModel.getState().setText(translatedText);
+        translateTextViewModel.getState().setText(inputText);
```

Comment:
actually translatedText is the output text, so all you need to do is remove inputText from the parameter. but for your state class, you need the instance attributes inputText and outputText as well as setters for them